eo-maven-plugin/src/main/java/org/eolang/maven/GmiMojo.java

```diff
@@ -374,18 +374,18 @@ private void makeGraph(final String xembly, final Path gmi) throws IOException {
                         .append(dirs)
                 ).domQuietly()
             );
-            new Save(
-                graph.toString(),
-                gmi.resolveSibling(String.format("%s.graph", gmi.getFileName()))
-            ).save();
+            new Home().save(
+                gmi.resolveSibling(String.format("%s.graph", gmi.getFileName())),
+                graph.toString()
+            );
             if (this.generateDotFiles) {
                 final String dot = new Xsline(GmiMojo.TO_DOT)
                     .pass(graph).xpath("//dot/text()").get(0);
                 Logger.debug(this, "Dot:\n%s", dot);
-                new Save(
-                    dot,
-                    gmi.resolveSibling(String.format("%s.dot", gmi.getFileName()))
-                ).save();
+                new Home().save(
+                    gmi.resolveSibling(String.format("%s.dot", gmi.getFileName())),
+                    new Xsline(GmiMojo.TO_DOT).pass(graph).xpath("//dot/text()").get(0)
```

Comment:
@yasamprom why did you copy it here? we already calculate `dot` a few lines above